OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -435,6 +459,10 @@ public void Tick(Actor self)
 
 			foreach (var b in builders)
 				b.Tick();
+
+			var ordersToIssueThisTick = Math.Min((int)Math.Ceiling((double)orders.Count / Info.MinOrderQuotientPerTick), orders.Count);
```

Comment:
`(orders.Count + Info.MinOrderQuotientPerTick - 1) / Info.MinOrderQuotientPerTick` is the equivalent integer calculation.